src/pipecat/frames/frames.py

```diff
@@ -558,6 +558,8 @@ class TTSUpdateSettingsFrame(ControlFrame):
     style: Optional[str] = None
     style_degree: Optional[str] = None
     role: Optional[str] = None
+    gender: Optional[str] = None
+    google_style: Optional[str] = None
```

Comment:
i'm not a fan of having such custom service settings in the base class, but given the current way things are setup, it seems like this is the best option. we should consider if there's a better way to handle custom setting updates in the future.

Reply:
I believe it's the same comment I posted. Would my proposal work?

Reply:
yeah, i think we were reviewing at the same time. I like your proposal and commented on how it might be fleshed out a bit more to maintain _some_ typing.